package-lock.json

```diff
@@ -19511,7 +19511,7 @@
 				},
 				"node-pre-gyp": {
 					"version": "0.12.0",
-					"resolved": "https://registry.npmjs.org/node-pre-gyp/-/node-pre-gyp-0.12.0.tgz",
+					"resolved": false,
```

Comment:
I don't see why these changes would be expected here?

When I run `npm install` on `master` using the recommended (latest) version of NPM (6.13.7), I see this being flipped back to the value it was previously.

Reply:
See a59a5d9ab761bb2e0ee37eb67aab96d7f3dcf91b